<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project visualization</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>321</width>
      <height>524</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LeftMenu">
     <item>
      <widget class="QLabel" name="lblBrs">
       <property name="text">
        <string>Branches</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="brComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblDevs">
       <property name="text">
        <string>Developers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="devComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="lblStartDate">
       <property name="text">
        <string>Start date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="startDateEdit">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEndDate">
       <property name="text">
        <string>End date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="endDateEdit">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVisualize">
       <property name="text">
        <string>Visualize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>148</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="activityView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>1580</width>
      <height>440</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1580</width>
      <height>440</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1580</width>
      <height>440</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="filesView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>411</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="commitsView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>281</width>
      <height>431</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCommits">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>148</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Commits</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFiles">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Files</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="diffView">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>190</y>
      <width>441</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblChanges">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>140</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenProject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open project</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveDevs">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save developers info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearchCommit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search for commit</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="lineChangeLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lines affected:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblLinesAffected">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Lines added:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Lines deleted:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblLinesDeleted">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblLinesAdded">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>364</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="authorLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="editAuthor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblEmail">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editEmail">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblAuthor">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>445</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="commitLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Commit:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editCommit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save developers info</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionLil">
   <property name="text">
    <string>lil</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
